client/actions/eventsPlanning/ui.js

```diff
@@ -313,6 +314,37 @@ const storeFilter = (filterId) => ({
     payload: filterId,
 });
 
+const fetchEventTemplates = () => (dispatch, getState, {api}) => {
+    api('recent_events_template').query()
+        .then((res) => {
+            dispatch({type: 'RECEIVE_EVENT_TEMPLATES', payload: res._items});
```

Comment:
Please use the constants when it comes to dispatching actions, i.e. `EVENTS.ACTIONS.RECEIVE_EVENT_TEMPLATES`